core/domain/stats_domain.py

```diff
@@ -616,12 +699,37 @@ def __init__(self, answer_occurrences=None):
             answer_occurrences if answer_occurrences else [])
 
     def to_raw_type(self):
+        """Returns the answer occurrences list with each answer represented as
+        a Python dict.
+
+        Returns:
+            list. A list containing answer occurrences dict in the following
```

Comment:
list(dict). A list of answer occurrence dicts. Each dict has the following format:

(and include types inline within the dict).

Reply:
Done!